src/components/LoginPage.js

```diff
@@ -7,15 +7,25 @@ import { useNavigate } from 'react-router-dom';
 const LoginPage = ({ onSuccessfulSubmit }) => {
   const [userCode, setUserCode] = useState('')
   const [agreed, setAgreed] = useState(false)
-  const secretCode = "test";
+  // const secretCode = 'test';
```

Comment:
I think you can just remove this line so the password can't be seen in plain text